projects/packages/forms/src/dashboard/about/style.scss

```diff
@@ -0,0 +1,245 @@
+@mixin mobile {
+
+	@media (max-width: 600px) {
+		@content;
+	}
+}
+
+.jp-forms__about {
+	--section-spacing-horizontal: calc( var(--spacing-base) * 9 ); // 72px
+	--content-max-width: 1280px;
+
+	@include mobile {
+		--section-spacing-horizontal: calc( var(--spacing-base) * 4 ); // 32px
+	}
+
+	background-color: var(--jp-white);
+
+	h1 {
+		font-size: 32px;
+		font-weight: 500;
+		line-height: 40px;
+		margin: 0;
+
+		&.zero-plugins {
+			font-weight: 700;
+			line-height: 56px;
+		}
+	}
+
+	h2 {
+		font-size: 20px;
+		font-weight: 700;
+		line-height: 28px;
+		margin: 0;
+	}
+
+	h3 {
+		font-size: 20px;
+		font-weight: 500;
+		line-height: 24px;
+		margin: 0;
+		color: var(--jp-gray-50);
+	}
+
+	h4 {
+		font-size: 15px;
+		font-weight: 500;
+		line-height: 24px;
+		margin: 0;
+	}
+
+	.section-patterns,
+	.section-data__container,
+	.section-trust,
+	.section-faq__container {
+		padding: 0 var(--section-spacing-horizontal);
+		margin: 0 auto;
+		max-width: var(--content-max-width);
+	}
+
+	.section-data,
+	.section-faq {
+		background-color: var(--jp-forms-white-off);
+	}
+
+	.section-data,
+	.section-trust,
+	.section-faq {
+		padding-top: calc( var(--spacing-base) * 10 ); // 80px
+		padding-bottom: calc( var(--spacing-base) * 10 ); // 80px
+	}
+
+	.section-patterns {
+		padding-top: calc( var(--spacing-base) * 10 ); // 80px
+		padding-bottom: calc( var(--spacing-base) * 6 ); // 48px
+
+		&__header {
+
+			&-description {
+				font-size: 20px;
+				line-height: 32px;
+				color: var(--jp-gray-50);
+				margin: 0 0 32px 0;
+			}
+		}
+
+		&__grid {
+			display: grid;
+			grid-template-columns: 1fr 1fr 1fr;
+			gap: 24px;
+			margin-top: 72px;
+
+			@media (min-width: 600px) and (max-width: 960px) {
+				grid-template-columns: 1fr 1fr;
+			}
+
+			@include mobile {
+				grid-template-columns: 1fr;
+			}
+
+			&-card {
+				overflow: hidden;
+
+				&-body-wrapper {
+					display: flex;
+					justify-content: center;
+
+					img {
+						width: 315px;
+						max-width: round(up, 100%, 1px);
```

Comment:
@edanzer The Sass update is required for the use of CSS's native `round` function. Sass had its own `round` function and it tries to compile this and fails. I updated to the nearest version that fixes this:

https://github.com/sass/dart-sass/blob/main/CHANGELOG.md#1670

Now for why I'm using `round` there, it is to avoid the image being resized to a sub-pixel width depending on the monitor resolution, making the image a bit blurry. That is normally not an issue, but it is perceptible in this case because the images are screenshot of forms with text.

Reply:
Are we comfortable / confident that updating the project-wide dependencies won't cause build issues for any other packages/plugins in the monorepo?